### DISCUSSION TITLE
Password managers compatibility

### DISCUSSION THREAD

**Discussion by nicoulaj** [goauthentik/authentik#2419] [Ideas]
Is there is anyway to use a simpler, Javascript-less login form template ?

1password does not autofill the current login and it is quite a usability issue for users. I filed a request to 1password [here](https://1password.community/discussion/127554/autofill-support-with-authentik).

As a side note, I think it would help if there was a publicly accessible demo instance of authentik.

**Comment by PKizzle** [comment]
This is also true for iCloud Keychain users and I guess many other password managers.

**Comment by Tuphal** [comment]
I finally took a look at this and got it working:

In the admin-GUI of authentik go to "Stages & Flows" -> "Flows" -> edit "default-authentication-flow" and enable "Compatibility mode".
Now it is working with 1Password in Firefox (Windows) and Safari (macOS and iOS)

Maybe a short information in the docs would be great for this hidden feature

**Reply by PKizzle** [reply]
Can confirm that it also works with iCloud Keychain. Why is this feature not enabled by default?

**Reply by jocelynthode** [reply]
Can confirm it also fixes it for Bitwarden.

**Reply by mspray** [reply]
Not working for username (bitwarden chrome plugin)

**Comment by deviantintegral** [comment]
Given how important password managers are to security, and that compatibility mode seems to be required for all password managers, I think this setting should be reverted to a “enable `<some name to describe the current behaviour>` mode”, and turned off by default. In other words, enable compatibility mode by default for all new flows.

**Comment by skavoori** [comment]
It’s almost 2026 and I don’t see why this is not default

**Comment by eringerli** [comment]
It's the past the middle of 2026 and I still had to search the internet to enable this very important functionality. Just make it the default, that would be a lot less headache.
